src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/ProfilePage.xaml

```diff
@@ -57,37 +57,37 @@
 						<Setter Target="MainGridViewColumn_2.Width"
 								Value="*" />
 						<Setter Target="MainStackPanelView.Margin"
-						        Value="55,40,0,0"/>
+								Value="55,40,0,0" />
 					</VisualState.Setters>
 				</VisualState>
 			</VisualStateGroup>
 		</VisualStateManager.VisualStateGroups>
 
 		<!-- PAGE HEADER -->
 		<controls:PageHeader x:Name="HeaderView"
-							 Grid.Column="0"
-							 Grid.ColumnSpan="2"
 							 Title="Profile"
 							 HamburgerMenuVisibility="Collapsed"
+							 Grid.Column="0"
```

Comment:
Having Grid.Row/Column at the top properties after x:Name is better in my opinion as opposed to having them last. Easier to read and understand the structure of the layout ;)